.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+
+env:
+  global:
+    - LUAJIT_LIB=/usr/lib64/libluajit-5.1.so
+    - LUAJIT_INC=/usr/include/luajit-2.0
+    - LUA_INCLUDE_DIR=/usr/include/luajit-2.0
```

Comment:
travis neither support Lua as a "language", not luajit-2.1 can be easily installed on trusty.
I'll think what can we do